gatsby-config.js

```diff
@@ -63,6 +63,11 @@ module.exports = {
               pubDate
             }
           }
+          allPrismicBook {
+            nodes {
```

Comment:
This is still failing the build, depending on your needs change here OR the `createPage` function